.github/workflows/ci.yml

```
@@ -79,7 +79,7 @@ jobs:
         with:
           fail-on: all
           sarif: results.sarif
-          slither-args: --filter-paths "./lib" --exclude naming-convention
+          slither-args: --filter-paths "./lib|./test" --exclude naming-convention
```

I think if you make this change:
- The `slither-analyze` CI check will always pass
- But the code scanning step should fail if there's findings
However I'm not certain of this so we'll need to test it

```suggestion
          slither-args: --no-fail-pedantic --filter-paths "./lib|./test" --exclude naming-convention
```

---

Actually wait, just two lines up you can change `fail-on` to none instead

---

Adjusted in aaa777d26138e0fe770391ddbbcc01d0a58a634c